Gruntfile.js

```diff
@@ -134,10 +134,10 @@ module.exports = function(grunt) {
 
   var browsers = [{
     browserName: "firefox",
-    version: "30"
+    platform: "linux"
   }, {
     browserName: "chrome",
-    version: "35"
+    platform: "linux"
```

Comment:
Did the change and there are more tests failing, due to pixel rounding differences (I believe)... I'd file another issue for this.